### DISCUSSION TITLE
RAP vs HRRR helicity variables switched?

### DISCUSSION THREAD

**Discussion by vc17b** [NOAA-GSL/GSL-Regional-Model-Forum#30] [Q&A] [answered]
From the RAP and HRRR (z-coord: pressure) tables, I see that the helicity variables for 0-1 km and 0-3 km are switched:

RAP (https://rapidrefresh.noaa.gov/GRIB2Table_NCEPrap_prs.txt):
- HLCY_P0_2L103_GLC0 **(1)** , Helicity, Storm-relative helicity for **0-1 km** above ground level 
- HLCY_P0_2L103_GLC0 **(0)** , Helicity, Storm-relative helicity for **0-3 km** above ground level 

HRRR (https://rapidrefresh.noaa.gov/hrrr/GRIB2Table_hrrrncep_2d.txt):
- HLCY_P0_2L103_GLC0 **(0)** , Helicity, Storm-relative helicity for **0-1 km** above ground level 
- HLCY_P0_2L103_GLC0 **(1)** , Helicity, Storm-relative helicity for **0-3 km** above ground level

I thought the level above ground for both of these were 0 = 1000 m and 1 = 3000 m. Let me know if this is correct, but I thought I would ask - it seems odd that it would be written one way for RAP but the opposite for HRRR.

**Comment by EricJames-NOAA** [comment, marked as answer]
Thanks for the question; that is indeed a typo.  The 0-3km HLCY is the first record for both the RAP and the HRRR.  The bottom line is to take care that you are using the one that you intend, since the NCL names are identical!

**Reply by vc17b** [reply]
Ok, so you mean the first record as labeled 0 for the level?

**Reply by EricJames-NOAA** [reply]
Correct.  The first record is indicated by the (0), which should be the 0-3km HLCY for both RAP and HRRR.
